<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVDBPipeEditWidget</class>
 <widget class="QWidget" name="SVDBPipeEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Insulation thickness:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QtExt::ColorButton" name="pushButtonColor">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Wall thermal conductivity:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditWallDensity"/>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>W/mK</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Wall roughness:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Category:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditWallThickness"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Insulation thermal conductivity:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>Wall density:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QtExt::LanguageStringEditWidget1" name="lineEditCategory"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Wall thickness:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="3">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>W/mK</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Effective conductivity per m pipe length:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditInsulationThickness"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Outer diameter:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditWallRoughness"/>
   </item>
   <item row="5" column="2">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditWallLambda"/>
   </item>
   <item row="8" column="3">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>W/mK</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelDisplayName">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QtExt::LanguageStringEditWidget1" name="lineEditName">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QLineEdit" name="lineEditUValue">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditInsulationLambda">
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditOuterDiameter"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>Wall specific heat capacity:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditWallHeatCapacity"/>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>kg/mÂ³</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QLabel" name="label_19">
     <property name="text">
      <string>J/kgK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtExt::ValidatingLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_ValidatingLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QtExt::ColorButton</class>
   <extends>QPushButton</extends>
   <header location="global">QtExt_ColorButton.h</header>
  </customwidget>
  <customwidget>
   <class>QtExt::LanguageStringEditWidget1</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_LanguageStringEditWidget1.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditName</tabstop>
  <tabstop>lineEditCategory</tabstop>
  <tabstop>lineEditOuterDiameter</tabstop>
  <tabstop>lineEditWallThickness</tabstop>
  <tabstop>lineEditWallRoughness</tabstop>
  <tabstop>lineEditWallLambda</tabstop>
  <tabstop>lineEditWallDensity</tabstop>
  <tabstop>lineEditWallHeatCapacity</tabstop>
  <tabstop>lineEditInsulationThickness</tabstop>
  <tabstop>lineEditInsulationLambda</tabstop>
  <tabstop>lineEditUValue</tabstop>
  <tabstop>pushButtonColor</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
